phabfive/maniphest.py

```diff
@@ -52,10 +53,24 @@ def create_from_config(self, config_file, dry_run=False):
             data = yaml.load(stream, Loader=yaml.Loader)
 
         #####
-        # STEP: Data structure validation, check title + description, projects existance
+        # STEP: Data structure validation, check title + description, projects, owner, subscribers existance
 
         ### Validate all projects is correct data types and exists in Phabricator instance
-        projects_query = self.phab.project.search(constraints={"name": ""})
+        #create a new construct to consolidate all project/tags
+        projects_query = {}
+        projects_query["data"] = []
+
+        r = self.phab.project.search(constraints={"name": ""})
```

Comment:
This comment is a general comment in all changes you submited in this PR. We generally avoid variable names less then 3 characters. Specially single letter characters. Variables should be easily understood from it's name
```suggestion
        r = self.phab.project.search(constraints = {"name": ""})
```